src/utils/order_watcher_utils.ts

```diff
@@ -0,0 +1,53 @@
+import { OrderEventEndState } from '@0x/mesh-graphql-client';
```

Comment:
I think we should remove it, explicitly import it and remove mesh dependencies altogether.